SOCIALCLUB/ScGamerdataGetInt.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: SOCIALCLUB
+aliases: ["_0xC85A7127E7AD02AA"]
+---
+## SC_GAMERDATA_GET_INT
+
+```c
+// 0xC85A7127E7AD02AA 0xF8BDA989
+BOOL SC_GAMERDATA_GET_INT(char *name, int *outValue);
+```
+
```

Comment:
This is missing a Parameters section since you added 2 new parameters, also the Return value.

Reply:
Added section